CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Fixed emoji `Icon` spacing issue and added settings `Icon` ([#991](https://github.com/stardust-ui/react/pull/991))
 - Call update `node` if it was changed for `Ref` component @layershifter ([#993](https://github.com/stardust-ui/react/pull/993))
 - Close previous `Popup` on enter key @jongsue ([#985](https://github.com/stardust-ui/react/pull/985))
+- Improve `Menu` styling, vertical menu in particular ([#934](https://github.com/stardust-ui/react/pull/934))
```

Comment:
Replace with: 
Improve `Menu` styling - vertical menu in Teams theme ([#934](https://github.com/stardust-ui/react/pull/934))

Reply:
Done.